rfcs/0013-avalanche.md

```diff
@@ -0,0 +1,98 @@
+# Replication via Avalanche 
```

Comment:
@sagar-solana i think we want to get away from using Avalance since there is a crypto project with the same name. 
 @garious what name should we use instead of avalanche?  Ledger Replication?

Reply:
how about the faberge protocol? https://www.youtube.com/watch?v=mcskckuosxQ

Reply:
@aeyakovenko, @sagar-solana, The book calls it *data plane fanout*. https://solana-labs.github.io/solana/cluster.html#confirming-transactions

But I'm not opposed to using the term "avalanche" as the networking algorithm that implements the data plane. That's consistent with how we use "gossip" as the networking algorithm that implements the control plane.